app/main.py

```diff
@@ -2,5 +2,16 @@
 
 
 def cache(func: Callable) -> Callable:
-    # Write your code here
-    pass
+    cash = {}
+
+    def inner(*args) -> int:
```

Comment:
Return type annotation is not correct because this decorator may be applied to functions that return values other than integers. Replace `int` with `Any`.